controllers/users.controller.js

```diff
@@ -0,0 +1,106 @@
+const { User } = require("../models/user.model");
+
+const createNewUser = async (req, res) => {
+  try {
+    const userData = req.body;
+    const user = await User.findOne({ email: userData.email });
+
+    if (user) {
+      res.status(409).json({
+        success: false,
+        message: "User Already exists with this email id. ",
+      });
+      return;
+    }
+
+    const NewUser = new User(userData);
```

Comment:
encrypting the password before storing into DB is a good practise

Reply:
Yes I will try to learn that